File: algo/src/test/java/com/hartwig/actin/algo/evaluation/tumor/HasLowRiskOfHemorrhageUponTreatmentTest.java
package com.hartwig.actin.algo.evaluation.tumor;

import static com.hartwig.actin.algo.evaluation.EvaluationAssert.assertEvaluation;

import com.hartwig.actin.TestDataFactory;
import com.hartwig.actin.algo.datamodel.EvaluationResult;

import org.junit.Test;

public class HasLowRiskOfHemorrhageUponTreatmentTest {

    @Test
    public void canEvaluate() {
        HasLowRiskOfHemorrhageUponTreatment function = new HasLowRiskOfHemorrhageUponTreatment();

        assertEvaluation(EvaluationResult.UNDETERMINED, function.evaluate(TestDataFactory.createMinimalTestPatientRecord()));
    }
}

File: algo/src/test/kotlin/com/hartwig/actin/algo/evaluation/tumor/HasLowRiskOfHemorrhageUponTreatmentTest.kt
package com.hartwig.actin.algo.evaluation.tumor

import com.hartwig.actin.TestPatientFactory
import com.hartwig.actin.algo.datamodel.EvaluationResult
import com.hartwig.actin.algo.evaluation.EvaluationAssert.assertEvaluation
import org.junit.Test

class HasLowRiskOfHemorrhageUponTreatmentTest {
    @Test
    fun canEvaluate() {
        val function = HasLowRiskOfHemorrhageUponTreatment()
        assertEvaluation(EvaluationResult.UNDETERMINED, function.evaluate(TestPatientFactory.createMinimalTestWGSPatientRecord()))
    }
}